lib/html_individual_config.dart

```diff
@@ -34,6 +34,8 @@ class HtmlIndividualConfiguration extends htmlconfig.HtmlConfiguration {
         }
 
         var testGroupName = groups.single.split('=')[1];
+        testGroupName =
+            testGroupName.replaceAll('+', ' ').replaceAll('%20', ' ');
```

Comment:
It seems like it would be much more robust to use `Uri.parse` and then accessing this via `Uri.queryParameters.`